src/data/reducers/index.js

```diff
@@ -3,16 +3,21 @@ import { routerReducer } from 'react-router-redux';
 import courseOutline from './courseOutline';
 import dashboardAnalytics from './dashboardAnalytics';
 import portalConfiguration from './portalConfiguration';
-import authentication from './authentication';
 import table from './table';
 import csv from './csv';
 
+const identityReducer = (state) => {
```

Comment:
Is there a reason why you're not having the `identityReducer` in its own file, i.e. `identityReducer.js`?

Reply:
Seemed like overkill in this case.